src/services-using.rst

```diff
@@ -69,15 +76,23 @@ Authenticating Users with QR-scanner
 
 Users can alternatively log in using the QR code scanner, with QR codes generated by the ODK-X app-designer
 
-   1. Press the Action Button (:guilabel:`⋮`) and select "Settings" options.
+   1. Click on the hamburger icon (:guilabel:`☰`) and select "Server Login" option.
+
+     .. image:: /img/services-using/services-server-login.*
+       :alt: Services Server Login Option
+       :class: device-screen-vertical
+   2. On the next screen, click on the :guilabel:`Scan QR` button.
+
+     .. image:: /img/services-using/server-setting-qr-scanner.*
+       :alt: Services login with QR code
+       :class: device-screen-vertical
 
-     .. image:: /img/services-using/services-options-settings.*
-   
-    1. Select :menuselection:`Settings --> Server Settings`
-    
-    2. Select the Scanner icon at the top right corner, to launch the QR-Scanner and scan the QR code generated by app-designer.
-    
-      .. image:: /img/services-using/server-setting-qr-scanner.*
+   3. The app will ask for camera permissions. Click :guilabel:`Allow`.
+   4. QR code scanner screen will appear. Scan a valid QR code.
+
+   .. note::
+
+     Use the web tab in the ODK-X Application Designer to generate QR codes.
```

Comment:
Is there a cross-link in the documentation that could be provided here?

Reply:
Done